mkosi/resources/mkosi.md

```diff
@@ -0,0 +1,1543 @@
+  explicitly specified.
+
+`shell`
+
+: This builds the image if it is not built yet, and then invokes
+  `systemd-nspawn` to acquire an interactive shell prompt in it. An optional
+  command line may be specified after the `shell` verb, to be invoked in place
+  of the shell in the container. Use `-f` in order to rebuild the image
+  unconditionally before acquiring the shell, see below. This command must be
+  executed as `root`.
+
+`boot`
+
+: Similar to `shell`, but boots the image using `systemd-nspawn`. An
+  optional command line may be specified after the `boot` verb, which
+  can contain extra nspawn options as well as arguments which are passed
+  as the "kernel command line" to the init system in the image.
+
+`qemu`
+
+: Similar to `boot`, but uses `qemu` to boot up the image, i.e. instead
+  of container virtualization virtual machine virtualization is used.
+  This verb is only supported for disk images that contain a boot loader
+  and cpio images in which a kernel was installed. For cpio images a
+  kernel can also be provided by passing the `-kernel` qemu argument to
+  the `qemu` verb. Any arguments specified after the `qemu` verb are
+  appended to the `qemu` invocation.
+
+`ssh`
+
+: When the image is built with the `Ssh=yes` option, this command
+  connects to a booted virtual machine (`qemu`) via SSH. Make sure to
+  run `mkosi ssh` with the same config as `mkosi build` was run with so
+  that it has the necessary information available to connect to the
+  running virtual machine via SSH. Any arguments passed after the `ssh`
+  verb are passed as arguments to the `ssh` invocation. To connect to a
+  container, use `machinectl login` or `machinectl shell`.
+
+`clean`
+
+: Remove build artifacts generated on a previous build. If combined
+  with `-f`, also removes incremental build cache images. If `-f` is
+  specified twice, also removes any package cache.
+
+`serve`
+
+: This builds the image if it is not built yet, and then serves the
+  output directory (i.e. usually `mkosi.output/`, see below) via a
+  small embedded HTTP server, listening on port 8081. Combine with
+  `-f` in order to rebuild the image unconditionally before serving
+  it. This command is useful for testing network based acquisition of
+  OS images, for example via `machinectl pull-raw …` and `machinectl
+  pull-tar …`.
+
+`bump`
+
+: Bumps the image version from `mkosi.version` and writes the resulting
+  version string to `mkosi.version`. This is useful for implementing a
+  simple versioning scheme: each time this verb is called the version is
+  bumped in preparation for the subsequent build. Note that
+  `--auto-bump`/`-B` may be used to automatically bump the version
+  after each successful build.
+
+`genkey`
+
+: Generate a pair of SecureBoot keys for usage with the
+  `SecureBootKey=`/`--secure-boot-key=` and
+  `SecureBootCertificate=`/`--secure-boot-certificate=` options.
+
+`documentation`
+
+: Show mkosi's documentation.
```

Comment:
Can you repeat that blurb from NEWS here as well? About generating the man page with pandoc and putting it in resources?